front/app/containers/Authentication/SuccessActions/actions/reactionOnIdea.ts

```diff
@@ -0,0 +1,49 @@
+import { IUserData } from 'api/users/types';
+import { addIdeaReaction } from 'api/idea_reactions/useAddIdeaReaction';
+import { deleteIdeaReaction } from 'api/idea_reactions/useDeleteIdeaReaction';
+import { queryClient } from 'utils/cl-react-query/queryClient';
+import ideasKeys from 'api/ideas/keys';
+
+export interface ReactionOnIdeaParams {
+  ideaId: string;
+  reactionId?: string | null;
+  reactionMode: 'up' | 'down';
```

Comment:
Maybe we can change this to `like` and `dislike`? Same in the `api/[idea|comment|initiative]_reactions/types`